doc/Development_Documentation/03_Documents/03_Navigation.md

```diff
@@ -487,6 +487,15 @@ $navigation->addPage([
    'title' => 'Homepage' //tooltip text
 ]);
 ```
+```twig
+ {% do mainNavigation.addPage({
+                            'order': -1,
+                            'uri': navStartNode.url,
+                            'label': 'Home',
+                            'title': "Home"
+                        })
+%}
```

Comment:
```suggestion
 {% do navigation.addPage({
                            'order': -1,
                            'uri': '/',
                            'label': 'Home',
                            'title': 'Homepage'
                        })
%}
```

Reply:
and please wrap them into a code-section container, like here: https://github.com/pimcore/pimcore/commit/19f4e0b9cf0713bfb55ff577b8c2c7e404f78d2c thanks!

Reply:
@dvesh3 the `'uri': '/'`is probably not the best solution when you have multiple languages...

Reply:
well, this is just a basic example and it does not talk about multiple languages setup here.

Reply:
@NORDCOMPCZ 🏓